packages/theme-sections/section.js

```diff
@@ -62,6 +64,10 @@ Section.prototype = {
     }
   },
 
+  _onLoad: function _onLoad(event) {
```

Comment:
I didn't include an `_onLoad()` method because it already exists as the `Section` constructor. `.onLoad()` is called on line 24.